recipes/libatomic_ops/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "7.4.2" %}
+
+package:
+  name: libatomic_ops
+  version: {{ version }}
+
+source:
+  fn: libatomic_ops-7.4.2.tar.gz
+  url: http://www.ivmaisoft.com/_bin/atomic_ops/libatomic_ops-7.4.2.tar.gz
+  sha1: 57cd7c64e37fca300bd7b24e3d2f14129b25b376
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+
+test:
+  commands:
+    - true
```

Comment:
I am not comfortable with this. Can we test for a library existence? See the recipe below for an example:

https://github.com/conda-forge/staged-recipes/pull/239/files#diff-efc5664ac8f8d3a512d291b45c29608dR29